tests/spec/core/markdown-spec.js

```diff
@@ -330,6 +330,22 @@ function getAnswer() {
     expect(pre.textContent).toBe(idl);
   });
 
+  it("does not parse texts inside <pre> as markdown", async () => {
+    const code = `1 * 2 * 3;`;
```

Comment:
Any reason to not inline this?

Reply:
This line is reused in line 346, IMO better as a separate value 👀